webapp-django/crashstats/signature/tests/test_views.py

```diff
@@ -843,3 +843,69 @@ def mocked_supersearch_get(**params):
 
         # Logged in users with the permission can see exploitability
         ok_('Exploitability' in response.content)
+
+    @mock.patch('requests.get')
+    def test_signature_summary_with_many_hexes(self, rget):
+
+        def mocked_get(url, params, **options):
+            if '/graphics_devices' in url:
+                ok_(len(params['vendor_hex']) <= 50)
+                ok_(len(params['adapter_hex']) <= 50)
+
+                return Response({
+                    'hits': [],
+                    'total': 0
+                })
+
+            raise NotImplementedError()
+
+        rget.side_effect = mocked_get
+
+        def mocked_supersearch_get(**params):
+            ok_('signature' in params)
+            eq_(params['signature'], ['=' + DUMB_SIGNATURE])
+
+            adapters = [
+                {
+                    'term': '0x{0:0>4}'.format(i),
```

Comment:
No, I knew it was a format trick. I just didn't recognize it "by heart". 
We only comment on business logic.